test/contracts/coin_toss.aes

```diff
@@ -0,0 +1,103 @@
+include "String.aes"
```

Comment:
This will not work on LIMA.

You need to put another contract in `test/contracts/sophia_4/` as discussed earlier.